### PR TITLE
namespaces: change team label of namespaces managed by CSA

### PR DESCRIPTION
**Pull request by daichimukai**
issue: cybozu/csa#81

### CODE REVIEW

File: rook/base/ceph-hdd/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ceph-hdd
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
```
Comment (yokaze) [inline rook/base/ceph-hdd/namespace.yaml:6]: Unlike SubNamespaces, Namespaces are prioritized by Argo CD automatically.

```suggestion
```

File: rook/base/ceph-object-store/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ceph-object-store
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
```
Comment (yokaze) [inline rook/base/ceph-object-store/namespace.yaml:6]: ditto
```suggestion
```

File: rook/base/ceph-ssd/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ceph-ssd
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
```
Comment (yokaze) [inline rook/base/ceph-ssd/namespace.yaml:6]: ditto
```suggestion
```

File: rook/poc/ceph-poc-2/namespace.yaml [hunk 4]
```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: ceph-poc-2
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
```
Comment (yokaze) [inline rook/poc/ceph-poc-2/namespace.yaml:6]: ditto
```suggestion
```

File: rook/poc/ceph-poc/namespace.yaml [hunk 5]
```diff
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: ceph-poc
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
```
Comment (yokaze) [inline rook/poc/ceph-poc/namespace.yaml:6]: ditto
```suggestion
```

File: test/setup_test.go [hunk 6]
```diff
@@ -408,6 +408,17 @@ func testSetup() {
 	})
 }
 
+func getRevisionOfApplication(appName string) string {
```
Comment (yokaze) [inline test/setup_test.go:411]: I prefer to use `getApplicationRevision`.

```suggestion
func getApplicationRevision(appName string) string {
```

**Review by yokaze — CHANGES_REQUESTED** [review]

File: rook/base/ceph-hdd/namespace.yaml (hunk 1 above)
Comment (daichimukai) [reply rook/base/ceph-hdd/namespace.yaml:6]: This namespaces hold config map with sync-wave: -1, so this annotation is necessary.

https://github.com/cybozu-go/neco-apps/blob/551b89abc21ec16a6f191eee54cb6fd3c65cb79d/rook/base/ceph-hdd/configmap.yaml#L5-L7

File: test/setup_test.go (hunk 6 above)
Comment (daichimukai) [reply test/setup_test.go]: Renamed.

**Review by yokaze — APPROVED** [review]
LGTM

**Pull request merged**